test/units/cli/galaxy/test_display_collection.py

```diff
@@ -8,14 +8,13 @@
 import pytest
 
 from ansible.cli.galaxy import _display_collection
+from ansible.galaxy.dependency_resolution.dataclasses import Requirement
 
 
 @pytest.fixture
 def collection_object(mocker):
```

Comment:
Since the `mocker` fixture is not used anymore, there's no need to inject it anymore:
```suggestion
def collection_object():
    """Return a requirement objects factory."""
```